packages/connect-query/package.json

```diff
@@ -51,7 +51,7 @@
   },
   "peerDependencies": {
     "@bufbuild/protobuf": "2.x",
-    "@connectrpc/connect": "^2.0.1",
+    "@connectrpc/connect": "^2.0.2",
```

Comment:
```suggestion
    "@connectrpc/connect": "^2.0.1",
```

Reply:
Hmm, actually, i wonder if we should just do `>=2.0.1`. I think then dependabot won't modify these.

Reply:
For now i'll just revert the upgrade here as well.